customers/schema/queries.py

```diff
@@ -211,21 +209,18 @@ class Query:
 
     @view_permission_required(CustomerProfile, BerthApplication, BerthLease)
     def resolve_berth_profiles(self, info, **kwargs):
-        berth_lease_customer = kwargs.pop("berth_lease_customer", False)
-        harbors = kwargs.pop("harbors", [])
-        piers = kwargs.pop("piers", [])
-        berths = kwargs.pop("berths", [])
-
-        marked_winter_storage_lease_customer = kwargs.pop(
-            "marked_winter_storage_lease_customer", False
-        )
-        marked_winter_storage_areas = kwargs.pop("marked_winter_storage_areas", [])
-        marked_winter_storage_places = kwargs.pop("marked_winter_storage_places", [])
-
-        unmarked_winter_storage_lease_customer = kwargs.pop(
-            "unmarked_winter_storage_lease_customer", False
-        )
-        unmarked_winter_storage_areas = kwargs.pop("unmarked_winter_storage_areas", [])
+        harbors = [h for h in kwargs.pop("harbors", []) if h is not None]
+        piers = [p for p in kwargs.pop("piers", []) if p is not None]
+        berths = [b for b in kwargs.pop("berths", []) if b is not None]
```

Comment:
I guess one way to prevent this would be to define the argument like `berths=graphene.List(graphene.NonNull(graphene.String))`. Though, would that break something. :thinking:

Reply:
Could be possible like that, but it would be useless without UI changes. UI should not send [None] either, but a None instead. Harbors, piers and berths would be checked the same way anyway. Quite trivial change in my opinion. The thing is that do we want the filters to be always included in the request.